torch/csrc/distributed/c10d/init.cpp

```diff
@@ -2845,7 +2964,8 @@ options :class:`~torch.distributed.ProcessGroupNCCL.Options`).
           .def(
               "abort",
               &::c10d::ProcessGroupNCCL::abort,
-              py::call_guard<py::gil_scoped_release>())
+              py::call_guard<py::gil_scoped_release>(),
+              R"(Abort the process group.)")
```

Comment:
note to check; do we have proper docs about this API in our c10d layer? iirc it is a new/experimental api

Reply:
I don't think we have any docs on abort